src/frame.c

```diff
@@ -65,11 +65,12 @@ ID3v2_frame_text_content* parse_text_frame_content(ID3v2_frame* frame)
     {
         return NULL;
     }
-    
+
     content = new_text_content(frame->size);
     content->encoding = frame->data[0];
-    content->size = frame->size - ID3_FRAME_ENCODING;
```

Comment:
Why this change?

Reply:
I changed that to fix an issue I have been having with mangled text. However, that solved the issue but not for some songs that I was testing with but not all. I'll revert it back to the way it was before.